SU-SOE/stanford_soe_helper
# SOE-2250: removed margin-bottom for hero banner at max 767.

# READY FOR REVIEW

# Summary
- This sets the margin-bottom to 0 at max 767 width

# Needed By (Date)
- Abracadabra sprint end

# Criticality
- Fixes broken stuff


# Steps to Test
- Switch to this branch
- May need to revert feature
- Navigate to /admission-aid
- Verify you see less margin bottom at width 767 and smaller
- Navigate to /admission-aid/financial-aid
- Verify it looks okay

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
Dev: https://stanfordits.atlassian.net/browse/SOE-2294
## Related PRs

## More Information
### Before screen shots
![bannerbefore01](https://user-images.githubusercontent.com/284440/31564940-dcfc2bfc-b019-11e7-982a-2b42720ebd25.png)
![bannerbefore02](https://user-images.githubusercontent.com/284440/31564952-e550e702-b019-11e7-92c4-c8f4f9bf6b59.png)

### After screenshots:
![bannerafter01](https://user-images.githubusercontent.com/284440/31564980-fc5f0b18-b019-11e7-8156-5de5e8469971.png)

![bannerafter02](https://user-images.githubusercontent.com/284440/31564927-cf604d02-b019-11e7-9180-95ec5f7a0f99.png)

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
